drivers/video/vnc/Kconfig

```diff
@@ -139,6 +139,21 @@ config VNCSERVER_KBDENCODE
 		Use a special encoding of keyboard characters as defined in
 		include/nuttx/input/kbd_coded.h.
 
+config VNCSERVER_TOUCH
+	bool "Enable touch input"
+	default n
+	select INPUT
+	select INPUT_TOUCHSCREEN
+	---help---
+		Use touchscreen based input driver
+
+config VNCSERVER_TOUCH_DEVNAME
+	string "Touch input device name"
+	default "/dev/vnctouch"
```

Comment:
look like we need use /dev/inputx like other driver to improve the compatibility.